### PR TITLE
Fabo/1337 page transaction

### PR DESCRIPTION
**Pull request by faboweb**
Closes #ISSUE

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added entries in `CHANGELOG.md` with issue # and GitHub username
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: test/unit/specs/components/wallet/PageTransactions.spec.js [hunk 1]
```diff
@@ -1,20 +1,222 @@
...
+  const addresses = [
+    `cosmos15ky9du8a2wlstz6fpx3p4mqpjyrm5ctpesxxn9`,
+    `cosmos1pxdf0lvq5jvl9uxznklgc5gxuwzpdy5ynem546`
+  ]
+  const validatorAddresses = [
+    `cosmosvaladdr15ky9du8a2wlstz6fpx3p4mqpjyrm5ctqzh8yqw`,
+    `cosmosvaladdr15ky9du8a2wlstz6fpx3p4mqpjyrm5ctplpn3au`,
+    `cosmosvaladdr15ky9du8a2wlstz6fpx3p4mqpjyrm5ctgurrg7n`
+  ]
+  const allTransactions = [
```
Comment (jbibla) [inline test/unit/specs/components/wallet/PageTransactions.spec.js]: @sabau has made the case that we should put this kind of data in the JSON folder

File: test/unit/specs/components/wallet/PageTransactions.spec.js [hunk 2]
```diff
@@ -48,15 +250,14 @@ describe(`PageTransactions`, () => {
     $store = {
       commit: jest.fn(),
       dispatch: jest.fn(),
-      getters
+      getters: JSON.parse(JSON.stringify(getters)) // clone so we don't overwrite by accident
```
Comment (jbibla) [inline test/unit/specs/components/wallet/PageTransactions.spec.js]: are the `.parse` and `.stringify` necessary?

Comment (faboweb) [reply test/unit/specs/components/wallet/PageTransactions.spec.js]: this is the safes and quickest way I know to clone an object

File: test/unit/specs/components/wallet/PageTransactions.spec.js (hunk 1 above)
Comment (sabau) [reply test/unit/specs/components/wallet/PageTransactions.spec.js]: I'm thinking to open a ticket about that because I'm spreading this around all PRs XD

Comment (jbibla) [reply test/unit/specs/components/wallet/PageTransactions.spec.js]: ok makes sense. @faboweb do you want to move this to a JSON file or shall we merge as is?

Comment (faboweb) [reply test/unit/specs/components/wallet/PageTransactions.spec.js]: I propose merging as is and do the refactor based on Karolys proposal later.

**Pull request merged**